core/templates/dev/head/pages/base.html

```diff
@@ -37,6 +37,8 @@
   <head>
     {% block prerender %}
     {% endblock prerender %}
+    <link rel="preload" href="/third_party/generated/webfonts/fa-solid-900.woff2" as="font" type="font/woff2">
```

Comment:
Hi @vojtechjelinek, better we keep it after pre-render?

Reply:
Sure